css/src/atomics/colors.scss

```diff
@@ -0,0 +1,225 @@
+	}
+
+	.background-color-#{$name}-tablet {
+		@include tablet {
+			outline-color: $invert;
+			background-color: $base !important;
+		}
+	}
+
+	.background-color-#{$name}-desktop {
+		@include desktop {
+			outline-color: $invert;
+			background-color: $base !important;
+		}
+	}
+
+	.background-color-#{$name}-widescreen {
+		@include widescreen {
+			outline-color: $invert;
+			background-color: $base !important;
+		}
+	}
+
+	.background-color-#{$name}-invert {
+		outline-color: $base;
+		background-color: $invert !important;
+	}
+
+	.background-color-#{$name}-light {
+		outline-color: $dark;
+		background-color: $background !important;
+	}
+
+	.background-color-#{$name}-dark {
+		outline-color: $background;
+		background-color: $dark !important;
+	}
+}
+
+// Text color
+
+.color-text {
+	color: $text !important;
+}
+
+.color-text-subtle {
+	color: $text-subtle !important;
+}
+
+.color-text-overlay-invert {
+	color: $overlay-invert;
+}
+
+.color-current-color {
+	color: currentColor !important;
+}
+
+.fill-current-color {
+	fill: currentColor !important;
+}
+
+.outline-color-text {
+	outline-color: $text !important;
+}
+
+.stroke-current-color {
+	stroke: currentColor !important;
+}
+
+// Background color
+
+.background-color-main {
+	outline-color: $text;
+	background-color: $body-background !important;
+}
+
+.background-color-main-tablet {
+	@include tablet {
+		outline-color: $text;
+		background-color: $body-background !important;
+	}
+}
+
+.background-color-main-desktop {
+	@include desktop {
+		outline-color: $text;
+		background-color: $body-background !important;
+	}
+}
+
+.background-color-main-widescreen {
+	@include widescreen {
+		outline-color: $text;
+		background-color: $body-background !important;
+	}
+}
+
+.background-color-main-medium {
+	outline-color: $text;
+	background-color: $body-background-medium !important;
+}
+
+.background-color-main-medium-tablet {
+	@include tablet {
+		outline-color: $text;
+		background-color: $body-background-medium !important;
+	}
+}
+
+.background-color-main-medium-desktop {
+	@include desktop {
+		outline-color: $text;
+		background-color: $body-background-medium !important;
+	}
+}
+
+.background-color-main-medium-widescreen {
+	@include widescreen {
+		outline-color: $text;
+		background-color: $body-background-medium !important;
+	}
+}
+
+.background-color-main-dark {
+	outline-color: $text;
+	background-color: $body-background-dark !important;
+}
+
+.background-color-main-dark-tablet {
+	@include tablet {
+		outline-color: $text;
+		background-color: $body-background-dark !important;
+	}
+}
+
+.background-color-main-dark-desktop {
+	@include desktop {
+		outline-color: $text;
+		background-color: $body-background-dark !important;
+	}
+}
+
+.background-color-main-dark-widescreen {
+	@include widescreen {
+		outline-color: $text;
+		background-color: $body-background-dark !important;
+	}
+}
+
+.background-color-alternate {
+	background-color: $body-background-alt !important;
```

Comment:
wouldn't the name here be `$alternate-background`, since body is tied to a variable?